### PR TITLE
[11.0] [MIG] sale_validity

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Default Quotation Validity
==========================

* With this module, you will be able to configure a default validity delay on quotations.

Migration of module Default Quotation Validity v10 to v11.

### CODE REVIEW

File: sale_validity/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+{
+    'name': 'Default Quotation Validity',
+    'version': '11.0.1.0.0',
+    'category': 'Sales',
+    'license': 'AGPL-3',
+    'summary': 'Set a default validity delay on quotations',
+    'depends': ['sale_management'],
+    'author': 'Camptocamp,Odoo Community Association (OCA)',
+    'website': 'http://www.camptocamp.com',
```
Comment (nikul-serpentcs) [inline sale_validity/__manifest__.py]: @murtuzasaleh Change URL as per https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Comment by pedrobaeza** [comment]
Can you squash a bit commit history avoiding that merge commits?

**Comment by murtuzasaleh** [comment]
> Can you squash a bit commit history avoiding that merge commits?

@pedrobaeza  squashed

**Review by JayVora-SerpentCS — APPROVED** [review]
LGTM

**Comment by pedrobaeza** [comment]
Can you please check conflicts? Why are you modifying that file?

**Comment by murtuzasaleh** [comment]
> Can you please check conflicts? Why are you modifying that file?

@pedrobaeza Done, Here I rebase PR then conflicts error occurs,  so I solved it.

**Comment by nikul-serpentcs** [comment]
@pedrobaeza @JayVora-SerpentCS Could you please review it and move Forward.

**Comment by LoisRForgeFlow** [comment]
@murtuzasaleh could you please fix conflicts?

**Comment by nikul-serpentcs** [comment]
@lreficent could you please review it?

**Comment by LoisRForgeFlow** [comment]
@murtuzasaleh that's not the way to solve the conflict (all those "Merge..." commits) you should do a rebase on top of OCA's 11.0 branch.

**Comment by murtuzasaleh** [comment]
@lreficent Conflicts solved! Could you please review it?

File: sale_validity/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (HviorForgeFlow) [inline sale_validity/__init__.py]: Remove encode

File: sale_validity/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```
Comment (HviorForgeFlow) [inline sale_validity/__manifest__.py]: Remove encode

File: sale_validity/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (HviorForgeFlow) [inline sale_validity/models/__init__.py]: Remove encode

File: sale_validity/models/company.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```
Comment (HviorForgeFlow) [inline sale_validity/models/company.py]: Remove encode

File: sale_validity/models/res_config_settings.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```
Comment (HviorForgeFlow) [inline sale_validity/models/res_config_settings.py]: Remove encode

File: sale_validity/models/sale_config_settings.py [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```
Comment (HviorForgeFlow) [inline sale_validity/models/sale_config_settings.py]: Remove encode

File: sale_validity/models/sale_order.py [hunk 8]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```
Comment (HviorForgeFlow) [inline sale_validity/models/sale_order.py]: Remove encode

File: sale_validity/tests/__init__.py [hunk 9]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (HviorForgeFlow) [inline sale_validity/tests/__init__.py]: Remove encode

File: sale_validity/tests/test_sale_validity.py [hunk 10]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```
Comment (HviorForgeFlow) [inline sale_validity/tests/test_sale_validity.py]: Remove encode

File: sale_validity/views/sale_config_settings.xml [hunk 11]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
```
Comment (HviorForgeFlow) [inline sale_validity/views/sale_config_settings.xml]: That view can be removed.

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Would be nice to use new [readme format](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)

**Comment by JayVora-SerpentCS** [comment]
@murtuzasaleh ping

**Review by LoisRForgeFlow — APPROVED** [review]
Functional review ok :+1:

**Comment by LoisRForgeFlow** [comment]
@hveficent could you update your review?

**Review by HviorForgeFlow — APPROVED** [review]
:+1:

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Odoo core already contains an expiration date:

seleccion_012

so you should reuse that field instead of creating a new one. And thus the module should be renamed to other thing like `sale_order_default_expiration`

**Review by pedrobaeza — APPROVED** [review]
I looked incorrectly to this and it's not creating a new field. I merge.

**Pull request merged**